src/SceneManager.ts

```diff
@@ -185,6 +185,12 @@ export class SceneManager {
     }
   }
 
+  public thirdPersonFollow(entityName: String): void {
```

Comment:
In typescript, the type should be `string`, all lowercase.

I see this is a persistent issue throughout the code, I'll open an issue to clean that.

Reply:
Yeah, I would suggest we do a separate pass/PR for this.  I'll point out that the ThreeJS documentation uses the uppercase, which is probably where this came from: https://threejs.org/docs/#api/en/core/Object3D.getObjectByName

Reply:
Ths particular case is something ThreeJS' documentation is not good at. Even the type "any" has a capital (which doesn't exist in Ts).

Again, this can be done in a different PR. I even created the issue to keep track of it.